packages/icons-svg/scripts/utils.mjs

```diff
@@ -2,7 +2,7 @@ import fs from 'fs-extra';
 import path from 'path';
 import { fileURLToPath } from 'url';
 
-export const __dirname = path.dirname(fileURLToPath(import.meta.url));
+export const __dirname = path.dirname(fileURLToPath(require.resolve(import.meta.url)));
```

Comment:
Var det meningen at denne endringen skulle med i PRen? Dette fungerer vel egentlig ikke, for require er ikke tilgjengelig i ESM?

Reply:
det gikk litt for i svingene igår